models.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel
+
+class TrainRequest(BaseModel):
+    pre_trained_model_path: str
+    dataset_config_path: str
+    epochs: int
+
+
+class TestRequest(BaseModel):
+    path_to_best_weights: str
+    path_to_test_data: str
```

Comment:
Instead of `path_to_test_data`, let's make it an image upload. For that you would need to use a Form (or FormData parameter)

Reply:
For that, we might have to make some changes in `test_model` as well.